### ISSUE TITLE
Implement display_help function

### ISSUE THREAD

**Issue by yeeunmariakim** [Accelegator/accelegator#19]
The starting file for this issue has been pushed to the branch `feature/i-19-display-help`. Checkout this branch instead of creating a new branch by running the command: `feature/i-19-display-help`.

This feature is to display all of the possible REPL commands and descriptions of their behavior. Here are some notes on each command:
![accelegator-planning-photo](https://user-images.githubusercontent.com/19576999/32033815-d611b52e-b9dc-11e7-805c-30f4a3ff74ee.jpg)

When the `display_help` function is called, something like this should print out.
```
Commands
--------
help                                    display commands and their brief descriptions
help <command>                          display verbose descriptions of <command> and valid arguments
show <email>                            display flattened responses for advisee with <email>

...
```

**Comment by karolvargas** [comment]
Added print line statements into display_help function

**Comment by yeeunmariakim** [comment]
@karolvargas I just took a look at your branch and I just want to clarify with you--this feature requires all of the commands listed on the board in the photo to be printed out. The example I gave included just some of the commands, but in order to complete this feature, the other commands (e.g. list, search, write) need print statements as well.
Also, I would recommend splitting each command into its own print statement, instead of using newlines in one print statement. Just for readability's sake :)

**Comment by karolvargas** [comment]
updated print statements but am a little confused as to what you want to be printed for the search field keyword command

**Comment by yeeunmariakim** [comment]
It should be something like "search given <field> of all responses of all students for keyword". Also, could you refactor your function to return one final string, instead of printing out the strings? You might also use Python's format function to properly align the strings.

**Comment by karolvargas** [comment]
Should I use triple quotes and set the whole block equal to a string value and then return the one string like how I did in the last push? or do you want me to refactor it differently?

**Comment by karolvargas** [comment]
refactored and properly indented help window

**Comment by karolvargas** [comment]
submitted to pull request

**Comment by karolvargas** [comment]
function is working fine but alignment of descriptions is slightly to the left

**Comment by yeeunmariakim** [comment]
Addressed in issue #53
